opta/core/aws.py

```diff
@@ -77,6 +78,23 @@ def delete_opta_config(self) -> None:
         if resp["ResponseMetadata"]["HTTPStatusCode"] != 204:
             raise Exception(f"Failed to delete opta config in {bucket}/{config_path}.")
 
+        s3_client = boto3.client("s3")
+        resp = s3_client.delete_object(Bucket=bucket, Key=self.layer.name)
+        if resp["ResponseMetadata"]["HTTPStatusCode"] != 204:
```

Comment:
Add a comment here to explain what this if implies?

Reply:
Check out new function